sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/AdministratorContact.java

```diff
@@ -1,108 +1,162 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
 
 package com.azure.security.keyvault.certificates.models;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
+import com.azure.core.annotation.Fluent;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
 
-/**
- * Represents an administrator in {@link CertificateIssuer}
- */
-public final class AdministratorContact {
-
-    /**
+/** Details of the organization administrator of the certificate issuer. */
+@Fluent
+public final class AdministratorContact implements JsonSerializable<AdministratorContact> {
+    /*
      * First name.
      */
-    @JsonProperty(value = "first_name")
     private String firstName;
 
-    /**
+    /*
      * Last name.
      */
-    @JsonProperty(value = "last_name")
     private String lastName;
 
-    /**
+    /*
      * Email address.
      */
-    @JsonProperty(value = "email")
     private String email;
 
-    /**
+    /*
      * Phone number.
      */
-    @JsonProperty(value = "phone")
     private String phone;
 
+    /** Creates an instance of AdministratorContact class. */
+    public AdministratorContact() {}
+
     /**
-     * Get the first name of the admin.
-     * @return the first name of admin.
+     * Get the firstName property: First name.
```

Comment:
I'll take this back to the Autorest crew on how this could be improved. I believe it was done this way to show the property being returned/modified and the description of said property.